sde-cookbook/qtcreator/configure-qtcreator.py

```diff
@@ -99,13 +99,13 @@ def add_qt_version(self):
                         + " --id " + self.qtversion_id
                         + " --type RemoteLinux.EmbeddedLinuxQt"
                         + " --qmake " + qmake
-                        )
+                       )
         self.__add_rollback_cmd("rmQt --id " + self.qtversion_id)
 
     def add_kit(self, use_qt=False):
         qt_params = ""
         if use_qt:
-            qt_params += " --qt " + self.qtversion_id + " --mkspec \"" + os.environ["QMAKESPEC"] + "\""
+            qt_params += ' --qt {} --mkspec "{}"'.format(self.qtversion_id, os.environ["QMAKESPEC"])
```

Comment:
By the way, I'm not sure this is PEP8 related.  
It's not really an important change, just felt the "\"" to be a little ugly...
Just say if you prefer the old one.

Reply:
I agree this is better